### PR TITLE
Ship InSpec 2

### PR DESCRIPTION
**Pull request by thommay**

### CODE REVIEW

**Comment by jquick** [comment]
Manhattan fixed with the Gemfile update:
http://manhattan.cd.chef.co/job/chef-trigger-ad_hoc/91/downstreambuildview/

File: kitchen-tests/.kitchen.travis.yml [hunk 1]
```diff
@@ -35,6 +35,7 @@ platforms:
     pid_one_command: /sbin/init
     intermediate_instructions:
       - RUN yum -y install sudo
+      - RUN yum -y install gcc-c++
```
Comment (tas50) [inline kitchen-tests/.kitchen.travis.yml]: Can you combine the 2 yum lines and add a comment on why we're adding gcc-c++. This reduces the number of layers that have to be created

File: kitchen-tests/.kitchen.travis.yml [hunk 2]
```diff
@@ -116,6 +121,8 @@ platforms:
     pid_one_command: /bin/systemd
     intermediate_instructions:
       - RUN /usr/bin/zypper install -y sudo
+      - RUN /usr/bin/zypper install -y gcc-c++
+
```
Comment (tas50) [inline kitchen-tests/.kitchen.travis.yml]: that extra line can go away while you're in there

**Comment by jquick** [comment]
Thanks @tas50. I think this is good to go.

**Pull request merged**